test/DynamoCoreWpfTests/DynamoTestUIBase.cs

```diff
@@ -83,6 +78,23 @@ public virtual void Start()
             SynchronizationContext.SetSynchronizationContext(new SynchronizationContext());
         }
 
+        /// <summary>
+        /// Derived test classes could override it to provide different 
+        /// configuration.
+        /// </summary>
+        /// <param name="settings"></param>
+        /// <returns></returns>
```

Comment:
Don't include the `params` and `returns` object at all if they're empty. Make the summary a complete sentence.